deps/v8/gypfiles/v8.gyp

```diff
@@ -2514,10 +2514,34 @@
         'GCC_ENABLE_CPP_RTTI': 'YES',        # -frtti
       },
       'defines': ['ANTLR4CPP_STATIC'],
+      'defines!': [
+        '_HAS_EXCEPTIONS=0',
+        'BUILDING_V8_SHARED=1',
+        'BUILDING_UV_SHARED=1',
```

Comment:
I guess these were the missing keys?  (Or rather, not missing...)

Reply:
(To figure this out I run the `GN` recipe and bisected the diff in the generated ninja files. That was not fun)
I guess `BUILDING_UV_SHARED` is inconsequential.